doc/docs/modules/ROOT/pages/reading.adoc

```diff
@@ -220,6 +220,32 @@ The created schema will be
 The returned column order is not guarantee to match the RETURN statement for Neo4j 3.* and Neo4j 4.0.
 Starting from Neo4j 4.1 it the order will be the same.
 
+[[bookmark-limit-query]]
+==== Limit the results
+
+Since partitioning is done by using LIMIT/SKIP you should not limit the result yourself.
```

Comment:
Mini rewrite of this section, to include new information I just discovered.

This connector does not permit using SKIP or LIMIT at the end of a Cypher query.  Attempts to do this will result in errors, such as the message "Please provide a valid READ query".

This is not supported because internally, the connector uses SKIP/LIMIT pagination to break read sets up into multiple partitions, to support partitioned reads.  As a result, user-provided SKIP/LIMIT clashes with what the connector itself adds to your query to support parallelism.

There is a work-around though; you can still accomplish the same by using SKIP/LIMIT internal inside of the query, rather than after the final RETURN block of the query.

Here's a simple example.   This first query will be rejected, and will fail:

```
MATCH (p:Person)
RETURN p.name as name
LIMIT 10
```

This same query though can be reformulated and will still work.

```
MATCH (p:Person)
WITH p.name as name LIMIT 10
RETURN p.name
```

The queries return the exact same data, but only the second one is usable with the spark connector, and partition-able.   If you choose to reformulate queries to use "internal SKIP/LIMIT" take careful notice of ordering operations to guarantee the same result set.

You may also use the `query.count` option rather than reformulating your query (rest of your text here)

Reply:
> This connector does not permit using SKIP or LIMIT at the end of a Cypher query. Attempts to do this will result in errors, such as the message "Please provide a valid READ query".

fyi now it says: `SKIP/LIMIT are not allowed at the end of the query`

changed the rest.